### PR TITLE
Chapter 9: editorial fixes

### PR DESCRIPTION
**Pull request by mlagally**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-architecture/pull/792.html" title="Last updated on Aug 4, 2022, 9:21 AM UTC (a9a9777)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-architecture/792/0b1117c...a9a9777.html" title="Last updated on Aug 4, 2022, 9:21 AM UTC (a9a9777)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3782,7 +3782,7 @@ <h2>Devices in a Local Network Controlled from a Cloud Service</h2>
           access all the <a>Things</a> behind the NAT device,
           assuming the local <a>Intermediary</a>
           has used a converged protocol that can traverse the NAT device
-          and has some way to expose this service to the Internet (STUN, TURN, DyDNS, etc.).
+          and has some way to expose this service to the Internet (STUN, TURN, DynDNS, etc.).
```
Comment (k-toumura) [inline index.html:3785]: "DynDNS" is the name of a specific company's Dynamic DNS service.
How about using "DDNS" or "Dynamic DNS"?

File: index.html [hunk 2]
```diff
@@ -3823,7 +3823,7 @@ <h2>Devices in a Local Network Controlled from a Cloud Service</h2>
         </figure>
         <p> In more difficult cases the NAT and firewall traversal may not work exactly as
           shown. In particular, an ISP may not support publicly accessible addresses,
-          or STUN/TURN and/or DyDNS may not be supported or available.
+          or STUN/TURN and/or DynDNS may not be supported or available.
```
Comment (k-toumura) [inline index.html:3826]: same as above.

File: index.html (hunk 1 above)
Comment (mlagally) [reply index.html]: done - used "Dynamic DNS"

File: index.html (hunk 2 above)
Comment (mlagally) [reply index.html]: done

**Pull request merged**